portfolio/src/main/webapp/js/script.js

```diff
@@ -53,9 +56,43 @@ async function getText() {
   }
 }
 
-/** Creates an <li> element containing text. */
-function createListElement(text) {
+/**
+ * Display text from data Servlet
+ */
+async function getText() {
+  const numComments = document.getElementById('number-comments');
+  getNumComments(numComments.options[numComments.selectedIndex].value);
+}
+
+/** Creates an <li> element containing text.
+ * @param {string} message A Json string representing a single comment.
+ */
+function createListElement(message) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  liElement.className = 'message';
+  liElement.innerText = message.text;
```

Comment:
if `message` is a string, what does `message.text` do? Should this not just be `liElement.innerText = message;`?

Reply:
Ah you're right, its actually the parsed JSON string, so its a JSON Object

Reply:
Are you sure `{JSON Object}` is a valid type? I think it might just be `{Object}` or even `{{text: string}}` to be more clear about the requirements.

https://jsdoc.app/tags-type.html